packages/table/src/common/_loading.scss

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+
+@import "../cell/common";
+
+@mixin skeleton-margin($base-class-name, $margin) {
+  .#{$base-class-name}-skeleton-#{$margin} {
+    margin-right: percentage($margin / 100);
+  }
+}
+
+@mixin skeleton-margins($base-class-name) {
+  @include skeleton-margin($base-class-name, 0);
+  @include skeleton-margin($base-class-name, 5);
+  @include skeleton-margin($base-class-name, 10);
+  @include skeleton-margin($base-class-name, 15);
+}
+
+@mixin skeleton($base-class-name) {
+  .#{$base-class-name}-skeleton {
+    flex: 1;
+    border-radius: 2px;
+    height: $pt-grid-size - 4px;
+    // negative animation delay ensures animations are synced
+    animation: 1.75s linear -7s infinite normal both running glow;
```

Comment:
1. Replace `1.75s` with `$pt-transition-duration * 20`
2. Seems pretty verbose for what it does. What about `animation: glow $pt-transition-duration infinite;`?